tensorflow_graphics/nn/layer/pointnet.py

```diff
@@ -208,8 +208,9 @@ def __init__(self, num_classes=40, momentum=.5, dropout_rate=.3):
     """
     super(PointNetVanillaClassifier, self).__init__()
```

Comment:
Feel free to rename this to VanillaClassifier, the fact that it is pointnet is inherited by the module name.
(as you are already changing stuff ;))